SonarQube.Client/Models/SonarQubeIssue.cs

```diff
@@ -41,5 +45,48 @@ public class SonarQubeIssue
         public string ModuleKey { get; }
         public string RuleId { get; }
         public bool IsResolved { get; }
+        public IReadOnlyList<IssueFlow> Flows { get; }
+    }
+
+    public class IssueFlow
+    {
+        private static readonly IReadOnlyList<IssueLocation> EmptyLocations = new List<IssueLocation>().AsReadOnly();
+
+        public IssueFlow(List<IssueLocation> locations)
+        {
+            Locations = locations ?? EmptyLocations;
+        }
+
+        public IReadOnlyList<IssueLocation> Locations { get; }
+    }
+
+    public class IssueLocation
+    {
+        public IssueLocation(string component, IssueTextRange textRange, string message)
+        {
+            Component = component;
+            TextRange = textRange;
+            Message = message;
+        }
+
+        public string Component { get; }
```

Comment:
Is `Component` = FilePath? `SonarQubeIssue` has a FilePath property, so I'd expect that to be the naming convention.

Reply:
Good catch.

Not exactly. Here,  `Component` is a composite data item in the form `[project key]:[relative file path]`. The code to create SonarQubeIssue instances has specific code to handle extracting the file path part. I'll create a separate PR for that.